### PR TITLE
Update to build frontend without Bazel

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by nghialv** [comment]
web-test fails since the Kapetanios config in the master is still being used.
So please ignore it for now.

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -51,12 +47,18 @@ jobs:
           key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
           restore-keys: |
             ${{ runner.os }}-yarn-
+      - name: Yarn install
+        working-directory: web
+        run: yarn install --prefer-offline
       - name: Build web static
-        run: make build-frontend
+        working-directory: web
+        run: yarn build
```
Comment (knanao) [inline .github/workflows/build.yaml:55]: What do you think about using the `make`command in CI in order to guarantee both commands of local and CI?.
```suggestion
        run: make web-build
```

Comment (nghialv) [reply .github/workflows/build.yaml]: You are right! I fixed them.

**Comment by knanao** [comment]
Nice!
/lgtm

**Comment by khanhtc1202** [comment]
Go bold 👍 
/approve

**Pull request merged**